### PR TITLE
Announce support for Python 3.7

### PR DESCRIPTION
**Pull request by florisla**
bump2version does not mention Python v3.7 support on PyPI, even though it works fine without any changes.

This commit adds the classifier `Programming Language :: Python :: 3.7` and tells Travis/Tox/AppVeyor to test against 3.7 too.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@ python:
   - "pypy"
   - "pypy3"
 
+# work-around for Python 3.7 which is on a non-standard Travis Ubuntu release (Xenial)
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```
Comment (ekohl) [inline .travis.yml]: Why does this need sudo?

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +9,13 @@ python:
   - "pypy"
   - "pypy3"
 
+# work-around for Python 3.7 which is on a non-standard Travis Ubuntu release (Xenial)
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
```
Comment (ekohl) [inline .travis.yml]: I don't mind moving the entire matrix to xenial. That should still support all the python versions.

**Comment by c4urself** [comment]
In future, we should also be bumping the Dockerfile as part of this, (this was put in place so people can run the test suite on all versions locally via `make test`). I've already added the change here: https://github.com/c4urself/bump2version/pull/52/commits/5f6d8f7261222c5ef5ab602dc398db4878f8cfec

File: .travis.yml (hunk 1 above)
Comment (florisla) [reply .travis.yml]: It's due to this Travis oddity: https://github.com/travis-ci/travis-ci/issues/10110 .

File: .travis.yml (hunk 2 above)
Comment (florisla) [reply .travis.yml]: Apparently Travis xenial does not support regular pypy / pypy3; I've kept those on trusty.  Relevant issue is https://travis-ci.community/t/pypy-2-7-on-xenial/889 .

**Comment by florisla** [comment]
I've documented the pypy-on-trusty and sudo-on-xenial oddities in `.travis.yml` itself.

**Pull request merged**